<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="menu_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="alg_menu">
      <item>
       <widget class="QPushButton" name="random_btn">
        <property name="text">
         <string>Random</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bfs_btn">
        <property name="text">
         <string>BFS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dfs_btn">
        <property name="text">
         <string>DFS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="greedy_btn">
        <property name="text">
         <string>Greedy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="astar_btn">
        <property name="text">
         <string>A*</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="img_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>550</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>171</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="open_label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Open: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="closed_label">
       <property name="text">
        <string>Closed: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="path_label">
       <property name="text">
        <string>Paht: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="speed_slider">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
